docs/introduction/Car-registry-tutorial.rst

```diff
@@ -490,7 +482,7 @@ As a first parameter we pass reference to SidechainNodeView, second reference is
             public String carSellOrderId; // hex representation of box id
             public long fee;
 
-            // Setters to let Akka jackson JSON library to automatically deserialize the request body.
+            // Setters to let Akka Jackson JSON library to automatically deserialize the request body.
```

Comment:
@zennermac 

            // Setters to let the Akka Jackson JSON library automatically deserialize the request body.